cmd/tckconvert.cpp

```diff
@@ -21,76 +21,83 @@
+    + Option ("scanner2voxel",
+        "if specified, the properties of this image will be used to convert "
+        "track point positions from real (scanner) coordinates into voxel coordinates.")
     + Argument ("reference").type_image_in ()
 
-  + Option ("scanner2image",
-      "if specified, the properties of this image will be used to convert "
-      "track point positions from real (scanner) coordinates into image coordinates (in mm).")
-  +    Argument ("reference").type_image_in ()
+    + Option ("scanner2image",
+        "if specified, the properties of this image will be used to convert "
+        "track point positions from real (scanner) coordinates into image coordinates (in mm).")
+    +    Argument ("reference").type_image_in ()
 
-  + Option ("voxel2scanner",
-      "if specified, the properties of this image will be used to convert "
-      "track point positions from voxel coordinates into real (scanner) coordinates.")
+    + Option ("voxel2scanner",
+        "if specified, the properties of this image will be used to convert "
+        "track point positions from voxel coordinates into real (scanner) coordinates.")
     + Argument ("reference").type_image_in ()
 
-  + Option ("image2scanner",
-      "if specified, the properties of this image will be used to convert "
-      "track point positions from image coordinates (in mm) into real (scanner) coordinates.")
-  +    Argument ("reference").type_image_in ()
+    + Option ("image2scanner",
+        "if specified, the properties of this image will be used to convert "
+        "track point positions from image coordinates (in mm) into real (scanner) coordinates.")
+    +    Argument ("reference").type_image_in ()
+
+    + OptionGroup ("Options specific to PLY writer")
 
-  + OptionGroup ("Options specific to PLY writer")
+    + Option ("sides", "number of sides for streamlines")
+    +   Argument("sides").type_integer(3,15)
 
-  + Option ("sides", "number of sides for streamlines")
-  +   Argument("sides").type_integer(3,15)
+    + Option ("increment", "generate streamline points at every (increment) points")
+    +   Argument("increment").type_integer(1)
 
-  + Option ("increment", "generate streamline points at every (increment) points")
-  +   Argument("increment").type_integer(1)
+    + OptionGroup ("Options specific to RIB writer")
 
-  + OptionGroup ("Options specific to RIB writer")
+    + Option ("dec", "add DEC as a primvar")
 
-  + Option ("dec", "add DEC as a primvar")
+    + OptionGroup ("Options for both PLY and RIB writer")
 
-  + OptionGroup ("Options for both PLY and RIB writer")
+    + Option ("radius", "radius of the streamlines")
+    +   Argument("radius").type_float(0.0f)
 
-  + Option ("radius", "radius of the streamlines")
-  +   Argument("radius").type_float(0.0f);
+    + OptionGroup ("Options specific to VTK writer")
+
+    + Option ("ascii", "write an ASCII VTK file (this is the default)")
```

Comment:
If one is the default, then a command-line option to activate such is redundant.